Ticket

```diff
@@ -31,6 +33,20 @@ public class Ticket extends TicketAbs {
     		return price_ar[movie][cinema][client.age][showtime];
     	else return -1;
     }
+    
+    public void create_ticket_price(double[][][][]price_ar) {
+    	//when there is a new movie, create the ticket pricings
+    	System.out.println("PLease enter movie name:");
+    	movie = sc.nextLine();
+    	System.out.println("Please enter cinema:");
+    	cinema = sc.nextLine();
+    	System.out.println("Please enter showtime:");
+    	showtime = sc.nextInt();
+    	System.out.println("Please enter ticket price for child:");
```

Comment:
Since in the document it's written that the ticket price is affected by factors like Age, Cinema type, Movie showtime etc, instead of taking the ticket price as input let's just set the initial ticket price automatically, based on the factors set above(we will call this method in the constructor when creating the ticket). For example, something along the lines of 

`if customer.age == child {
     ticket_price = movie.ticket_price[showtime] - movie.ticket_price[showtime]*0.2. //20% discount for child`